upstream-tag-sync/package.json

```
@@ -32,6 +31,6 @@
     "format:check": "prettier --check \"src/**/*.ts\"",
     "lint:staged": "lint-staged",
     "type-check": "tsc --noEmit",
-    "prepare": "husky"
+    "prepare": "git config core.hooksPath .githooks"
```

This PR doesn't change it but IMO we should rename it to something more explicit, “prepare” sounds like something we would do before e.g. doing a build rather than enabling git hooks for git operations.

Maybe something like this?

```suggestion
    "install-git-hooks": "git config core.hooksPath .githooks"
```

Or something more generic like this if you expect more steps to be added in this script command:

```suggestion
    "dev:setup": "git config core.hooksPath .githooks"
```

---

@devnoname120 Same as below.